src/app/views.py

```diff
@@ -202,10 +310,23 @@ def createIssue(licenseAuthorName, licenseName, licenseIdentifier, licenseCommen
     title = 'New license request: ' + licenseIdentifier + ' [SPDX-Online-Tools]'
     payload = {'title' : title, 'body': body, 'labels': ['new license/exception request']}
     headers = {'Authorization': 'token ' + token}
-    url = TYPE_TO_URL[urlType]
+    url = TYPE_TO_URL_LICENSE[urlType]
+    r = post(url, data=dumps(payload), headers=headers)
+    return r.status_code
+
+
+def createLicenseNamespaceIssue(licenseNamespace, token, urlType):
```

Comment:
This function can be defined in the `utils` file and then imported in the `views` file